samples/ClassroomLabs/Modules/Library/Tools/Readme.md

```diff
@@ -1,20 +1,138 @@
+  - [Show a menu to the user asking to select a lab and the value of some properties(Examples/PickLabAndProperties.ps1)](#show-a-menu-to-the-user-asking-to-select-a-lab-and-the-value-of-some-propertiesexamplespicklabandpropertiesps1)
+- [Structure of the example Hogwarts.csv](#structure-of-the-example-hogwartscsv)
+- [Structure of the example Charms.csv](#structure-of-the-example-charmscsv)
 
 ## Getting Started
-The [Bulk Lab Creation script](https://github.com/Azure/azure-devtestlab/tree/master/samples/ClassroomLabs/Modules/Library/Tools) is a stand-alone script that can be run from an authenticated Azure PowerShell session and requires [PowerShell](https://github.com/PowerShell/PowerShell/releases) and the [Azure PowerShell](https://docs.microsoft.com/en-us/powershell/azure/) module.  The script will automatically install the [ThreadJob](https://docs.microsoft.com/en-us/powershell/module/threadjob)  Powershell Module.
+The [Bulk Lab Creation functions](https://github.com/Azure/azure-devtestlab/tree/master/samples/ClassroomLabs/Modules/Library/Tools) can be run from an authenticated Azure PowerShell session and requires [PowerShell](https://github.com/PowerShell/PowerShell/releases) and the [Azure PowerShell](https://docs.microsoft.com/en-us/powershell/azure/) module.  The script will automatically install the [ThreadJob](https://docs.microsoft.com/en-us/powershell/module/threadjob)  Powershell Module.
+
+To get started, using the example configuration csv files:
+
 1. Get a local copy of the LabCreator.ps1 script by either [cloning the repo](https://github.com/Azure/azure-devtestlab.git) or by [downloading a copy](https://raw.githubusercontent.com/Azure/azure-devtestlab/master/samples/ClassroomLabs/Modules/Library/Tools/LabCreator.ps1)
 1. Get a local copy of the example [hogwarts.csv](https://raw.githubusercontent.com/Azure/azure-devtestlab/master/samples/ClassroomLabs/Modules/Library/Tools/hogwarts.csv) file and example [charms.csv](https://raw.githubusercontent.com/Azure/azure-devtestlab/master/samples/ClassroomLabs/Modules/Library/Tools/charms.csv) file
 1. Launch a PowerShell session
 1. Ensure [Azure PowerShell](https://docs.microsoft.com/en-us/powershell/azure/install-az-ps) installed
-1. Update the example CSV files to configure the resources to be created.  For addiitonal labs, create additional lines in the CSV file.  The CSV files can be modified diretly with Microsoft Excel.
-1. Execute the Bulk Lab Creation script with the following commandline:
+1. Update the example CSV files to configure the resources to be created.  For additional labs, create additional lines in the CSV file.  The CSV files can be modified directly with Microsoft Excel.
+1. Import the functions in the script by typing:
+```powershell
+Import-Module LabCreationLibrary.psm1
```

Comment:
Should this be "Import-Module .\LabCreationLibrary.psm1" ?

Reply:
Fixed in 24bba36bcd853608f67deaf98aba1555facf03ed